### PR TITLE
Milestone-2: Mobile First Portfolio pull request

### PR DESCRIPTION
**Pull request by HussenYesufAlli**
In this milestone, I: 
Use the HTML&CSS linter configuration 
Add header/Top bar section
Add Headline section

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+  <body>
+    <!-- The toolbar (or header) section -->
+
+    <header class="topBar-nav">
+      <div class="logo">
+        <a href="#"><img src="img/My-Logo.png" alt="HY" /></a>
+      </div>
+      <div class="icon-Union">
+        <a href="#"><img src="img/Union.png" alt="Three Dashs" /></a>
+      </div>
```
Comment (Qoosim) [inline index.html]: - [ ] It is required that the hamburger should be wrapped in a `nav` tag. Kindly change the ordinary `div` to `nav` tag. The nav tag is for semantics reason, and the right position for convention and user experience for mobile devices.

File: index.html [hunk 2]
```diff
@@ -0,0 +1,66 @@
...
+            </p>
+          </div>
+
+          <div class="lets-connect">
+            <a href="#">LET’S CONNECT</a>
+          </div>
+          <ul class="social-media-icons">
+            <li class="list">
+              <a href="https://twitter.com/">
+                <img src="img/t.png" alt="tiwetter" />
```
Comment (Qoosim) [inline index.html:47]: - [ ] Kindly reduce the font-size of this social media icon. It's kind of bigger than others.

### Your Design
your-design

### Figma Design
temp-design

**Review by Qoosim — CHANGES_REQUESTED** [review]
#  *Status: Required Changes* :recycle:

Hi @HussenYesufAlli, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  No linter errors. :heavy_check_mark: 
-  Good README file. :heavy_check_mark: 
-  Correct GiHub Flow. :heavy_check_mark: 




## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by AdnanOlarmmi — APPROVED** [review]
Hi @HussenYesufAlli ,

This is Adnan Is-haq, your beloved code reviewer, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 

Approved office man

## Highlights
- fixed previous reviewer suggestion.
- Nice UI

## Optional suggestions
_Check the comments under the review._

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**